packages/components/typography/src/Caption/Caption.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { FontWeightTokens } from '@contentful/f36-tokens';
+import type {
+  CommonProps,
+  MarginProps,
+  PolymorphicComponent,
+  PolymorphicProps,
+  ExpandProps,
+} from '@contentful/f36-core';
+import { Text } from '../Text';
+
+const CAPTION_DEFAULT_TAG = 'span';
```

Comment:
> `Caption` is used to display secondary and tertiary information

if it's mainly tend to live standalone of sentences and paragraphs, I'd recommend `strong` for better semantics

Reply:
Why is that? `strong` is defined like this by Mozilla: "The `<strong>` HTML element indicates that its contents have strong importance, seriousness, or urgency". We want to use `Caption` to de-emphasise information.

Reply:
Would you say `<small />` fits the purpose? I just think span is meant to wrap inline text within text and not to display a standalone info

Reply:
I don't think `small` has semantic meaning either, does it? To be honest I think `span` is perfect here because it has no semantic meaning, which I don't think we should add. Visually de-emphasising with just CSS makes sense to me.